pkg/capabilities/pb/capabilities_helpers_test.go

```diff
@@ -93,13 +93,10 @@ func TestMarshalUnmarshalRequest(t *testing.T) {
 	unmarshaled, err := pb.UnmarshalCapabilityRequest(raw)
 	require.NoError(t, err)
 
-	req.Metadata.ReferenceID = anyReferenceId
+	req.Metadata.ReferenceID = anyReferenceID
 	raw, err = pb.MarshalCapabilityRequest(req)
 	require.NoError(t, err)
 
-	unmarshaled, err = pb.UnmarshalCapabilityRequest(raw)
-	require.NoError(t, err)
-
```

Comment:
Changes in this file only because linter flagged these as issues. Fixing them made sense to me 🤷‍♂️